docs/INCIDENT.md

```
@@ -0,0 +1,250 @@
+우리 시스템이 처음 본 응답 → exception classification 보강 필요 (별도 issue).
+
+### 22:43:50 ~ 22:43:55 — Mail bombs
+
+외부 시스템이 user-X 의 등록된 휴대폰 + 이메일에:
+- "새로운 환경에서 로그인이 감지되었습니다" 알림 **5통** (5개 IP 가 외부에서 보안 임계값 초과한 것)
+
+Customer A 휴대폰에 22:43:50 부터 1초 간격으로 알림 5번.
+
+### 22:48:00 — Customer realizes
+
+Customer A 가 batch 결과 확인 → 모든 답글 실패 표시 → "내가 해킹당했나?" 의심.
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)